_includes/nav-links.html

```diff
@@ -1,11 +1,13 @@
 <a class="navbar-item blog {% if (page.url contains 'blog' %} active-tab {% endif %}"
   href="/blog">Blog</a>
+<a class="navbar-item meetup {% if page.url contains 'meetup' %} active-tab {% endif %}"
+  href="/meetup">Meetup</a>
 <a class="navbar-item learn {% if page.url contains 'learn' %} active-tab {% endif %}" id="learn"
   href="/learn">Learn</a>
 <a class="navbar-item meetup"
```

Comment:
Remove this meetup tag as it's duplicated

Reply:
oh ! okay seems it came during resolve conflict.